bindings/python/src/ser.rs

```diff
@@ -86,6 +87,10 @@ pub fn get_object_type(object_type: *mut pyo3::ffi::PyTypeObject) -> ObjectType
         ObjectType::Tuple
     } else if object_type == unsafe { types::DICT_TYPE } {
         ObjectType::Dict
+    } else if unsafe { (*(object_type as *mut ffi::PyTypeObjectSubObType)).ob_type }
```

Comment:
Maybe we can put this check into a separate function (with `#[inline]`)? E.g. `is_enum_subclass`